lib/matplotlib/tests/test_mathtext.py

```diff
@@ -237,7 +246,7 @@ def test_mathtext_rendering_svgastext(baseline_images, fontset, index, text):
     mpl.rcParams['svg.fonttype'] = 'none'  # Minimize image size.
     fig = plt.figure(figsize=(5.25, 0.75))
     fig.patch.set(visible=False)  # Minimize image size.
-    fig.text(0.5, 0.5, text,
+    fig.text(0.5, 0.5, text, fontsize=16,
```

Comment:
Note, I increased the size here because the existing test with dashes is quite small when converted to PNG (maybe half a pixel high, which is antialiased to two grey pixels).